### ISSUE TITLE
Can't add worklog

### ISSUE THREAD

**Issue by carlostrianag** [floralvikings/jira-connector#106]
Good night!

This is not an issue but a question :/

I'm trying to add a worklog to an issue using the following code:
```js
jira.issue.addWorkLog({
    "issueId": issue.id,
    "comment": "Testing tempo worklog.",
    "visibility": {
        "type": "group",
        "value": "jira-developers"
    },
    "started": "2018-07-13T00:35:37.095+0000",
    "timeSpentSeconds": 3600,
    "timeSpent": "1h"
}, function (error, issue) {
    if (error) {
        console.log("Failed to log work the ticket.");
        console.log(error);
    } else {
        console.log("No error");
    }
    process.exit(1)
});
```

And this is producing the following error:
```sh
node_modules/jira-connector/api/issue.js:889
            throw new Error(errorStrings.NO_WORKLOG_ERROR);
            ^

Error: Missing 'worklog' property
    at IssueClient.addWorkLog (/Users/carlostriana/Documents/mysaic_ruby/.git/hooks/node_modules/jira-connector/api/issue.js:889:19)
    at /Users/carlostriana/Documents/mysaic_ruby/.git/hooks/commit-msg:71:20
    at Request._callback (/Users/carlostriana/Documents/mysaic_ruby/.git/hooks/node_modules/jira-connector/index.js:368:24)
    at Request.self.callback (/Users/carlostriana/Documents/mysaic_ruby/.git/hooks/node_modules/request/request.js:185:22)
    at Request.emit (events.js:182:13)
    at Request.<anonymous> (/Users/carlostriana/Documents/mysaic_ruby/.git/hooks/node_modules/request/request.js:1157:10)
    at Request.emit (events.js:182:13)
    at IncomingMessage.<anonymous> (/Users/carlostriana/Documents/mysaic_ruby/.git/hooks/node_modules/request/request.js:1079:12)
    at Object.onceWrapper (events.js:273:13)
    at IncomingMessage.emit (events.js:187:15)
```

What should I send on "worklog" parameter? If I send whatever in there I got another error from JIRA.

Please help!

**Comment by floralvikings** [comment]
This repository is only _very_ lightly maintained; you are welcome to submit a pull request if there is behavior you'd like to add, I do occasionally check the PR list and merge code that is within the scope of this tool and doesn't include breaking changes.  

If you have found a bug please indicate the desired behavior and provide a minimal example demonstrating the incorrect behavior; I do try to fix bugs as they come up but I have very little time to devote to this repository anymore, so I can't make any guarantees.

If you have a feature you'd like to request, please indicate the desired behavior.  I will almost certainly never have the time to implement it myself but this repository does see occasional PR activity so it may be added by a generous contributor.  You are, of course, welcome to submit a pull request yourself implementing the desired behavior.

**Comment by MrRefactoring** [comment]
@carlostrianag good catch!

You're right, `worklog` is no longer such as property
 I fixed it in PR.
Changes will be in the next update.

Thanks for the issue

**Comment by MrRefactoring** [comment]
Fixed in 2.14.0 version

**Comment by gioboa** [comment]
@MrRefactoring 
Hi, I updated my project with jira-connector 2.15.3. 
In version 2.10.0 `issue.addWorklog` works like a charm.
Now with the new version I had this errors: 
- `Unrecognized field \"worklog\"` 
- `"Worklog must not be null."`
Maybe I miss somethings.

**Comment by MrRefactoring** [comment]
It seems that the worklog property is still used. Add it back

**Comment by MrRefactoring** [comment]
> @MrRefactoring
> Hi, I updated my project with jira-connector 2.15.3.
> In version 2.10.0 `issue.addWorklog` works like a charm.
> Now with the new version I had this errors:
> 
> * `Unrecognized field \"worklog\"`
> * `"Worklog must not be null."`
> 
> Maybe I missed some things.

@gioboa 

Fix it in version 2.15.4. 

Now I made it so that instead of `issue.addWorklog({worklog: {someProp: ''}})` you can now write `issue.addWorklog({someProp: ''})`

**Comment by gioboa** [comment]
Thanks :+1:
